src/ngraph/frontend/onnx_import/op/constant.cpp

```diff
@@ -0,0 +1,115 @@
+*******************************************************************************/
+
+#pragma once
+
+#include "ngraph/op/constant.hpp"
+#include "ngraph/frontend/onnx_import/node.hpp"
+#include "ngraph/frontend/onnx_import/tensor.hpp"
+#include "ngraph/node_vector.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace
+            {
+                template <typename T>
+                inline std::shared_ptr<ngraph::op::Constant>
+                    __make_ng_constant(const element::Type& type, const Tensor& tensor)
+                {
+                    return std::make_shared<ngraph::op::Constant>(
+                        type, tensor.get_shape(), tensor.get_data<T>());
+                }
+
+                template <Tensor::Type>
+                inline std::shared_ptr<ngraph::op::Constant> make_ng_constant(const Tensor& tensor)
+                {
+                    throw error::tensor::unsupported_data_type{tensor};
+                }
+
+                template <>
+                inline std::shared_ptr<ngraph::op::Constant>
+                    make_ng_constant<Tensor::Type::float16>(const Tensor& tensor)
+                {
+                    return __make_ng_constant<float>(element::f32, tensor);
+                }
+
+                template <>
+                inline std::shared_ptr<ngraph::op::Constant>
+                    make_ng_constant<Tensor::Type::float32>(const Tensor& tensor)
+                {
+                    return __make_ng_constant<float>(element::f32, tensor);
+                }
+
+                template <>
+                inline std::shared_ptr<ngraph::op::Constant>
+                    make_ng_constant<Tensor::Type::float64>(const Tensor& tensor)
+                {
+                    return __make_ng_constant<double>(element::f64, tensor);
+                }
+
+                template <>
+                inline std::shared_ptr<ngraph::op::Constant>
+                    make_ng_constant<Tensor::Type::int32>(const Tensor& tensor)
+                {
+                    return __make_ng_constant<int32_t>(element::i32, tensor);
+                }
+
+                template <>
+                inline std::shared_ptr<ngraph::op::Constant>
+                    make_ng_constant<Tensor::Type::uint32>(const Tensor& tensor)
+                {
+                    return __make_ng_constant<uint32_t>(element::u32, tensor);
+                }
+
+                template <>
+                inline std::shared_ptr<ngraph::op::Constant>
+                    make_ng_constant<Tensor::Type::uint64>(const Tensor& tensor)
+                {
+                    return __make_ng_constant<uint64_t>(element::u64, tensor);
+                }
+
+                inline std::shared_ptr<ngraph::op::Constant> make_constant(const Tensor& tensor)
+                {
+#define _make_ng_constant(_data_type)                                                              \
```

Comment:
`_make_ng_constant` to `MAKE_NG_CONSTANT`

Reply:
Done.